### PR TITLE
Update to Highline 2.1.0

### LINKED ISSUE #181: Upgrade highline gem
highline 1.6.21 has an LGPL file shipped within, which creates a hassle for license scanning.  The newer versions do not have this file, so let's see if we can upgrade it.

### PR DESCRIPTION
**Pull request by kbrock**
steal anything you want out of here @bdunne 

not thrilled with the changes to tests with the expect heard stuff.
wish the questions were not going through, but this is the least invasive way I could think to solve

overlaps with #200 

Fixes #181

### CODE REVIEW

File: spec/database_admin_spec.rb [hunk 1]
```diff
@@ -102,7 +102,7 @@
           expect(subject.ask_file_location).to be_truthy
 
           error = "Please provide #{errmsg}"
-          expect_heard ["Enter the #{prmpt}: ", error, prompt]
+          expect_heard ["Enter the #{prmpt}: |#{"/tmp/evm_db.backup"}| ", error, prompt]
```
Comment (Fryguy) [inline spec/database_admin_spec.rb:105]:
```suggestion
          expect_heard ["Enter the #{prmpt}: |/tmp/evm_db.backup| ", error, prompt]
```

**Comment by kbrock** [comment]
update:
- fixed highline gem value
- removed string interpolation in test
- moved dependency to spot where dependencies were defined (it was not finding the files for some reason, but it may have been another problem. seemed good to me anyway)

I tested on a nightly appliance. had to muck with the appliance console gemspec in 2 places and the vmdb/gemfile.lock, but I was able to setup dhcp, create a v2 key, set the region and configure an internal database.

**Comment by kbrock** [comment]
@bdunne let me know if you are happy with these changes.
as I mentioned I was able to change dhcp, and setup a basic appliance. If there are other configuration options you want me to test, let me know.

**Comment by kbrock** [comment]
@Fryguy or @bdunne I did run through those few tests. Anything else we want to do to be comfortable merging?

**Review by Fryguy — APPROVED** [review]
LGTM - @bdunne Please also review.

**Comment by Fryguy** [comment]
@kbrock Can you run the full rspec suite locally?  We still don't have github actions working in this repository yet.

**Comment by Fryguy** [comment]
Bouncing now that GitHub Actions is in place - 3.0 should still fail cause of kwargs at the moment

**Comment by kbrock** [comment]
update:
- removed the pending
update:
- removed the CI tty hack (still needed)
update
- testing out alternative no-echo input

**Review by Fryguy — APPROVED** [review]
LGTM

**Comment by Fryguy** [comment]
Love that you could kill the tty hack too

**Pull request merged**